packages/workflow/src/internals.ts

```diff
@@ -377,35 +384,75 @@ export class State {
    */
   public readonly signalHandlers = new Map<string, WorkflowSignalType>();
 
+  /**
+   * Source map file for looking up the source files in response to __enhanced_stack_trace
+   */
+  public sourceMap: RawSourceMap | undefined;
+
+  /**
+   * Whether or not to send the sources in enhanced stack trace query responses
+   */
+  public showStackTraceSources = false;
+
+  protected getStackTraces(): Stack[] {
+    const { childToParent, promiseToStack } = (globalThis as any).__TEMPORAL__.promiseStackStore as PromiseStackStore;
+    const internalNodes = new Set(
+      [...childToParent.values()].reduce((acc, curr) => {
+        for (const p of curr) {
+          acc.add(p);
+        }
+        return acc;
+      }, new Set())
```

Comment:
There are two `new Set()` in this. Looks like an incomplete refactor.

Reply:
Hmm.. I missed this, will fix, thanks!